helpers/error_handler/error_message.go

```diff
@@ -44,6 +46,8 @@ func DatabaseErrorMessage(err error) error {
 	pgErr := err.(*pgconn.PgError)
 
 	switch pgErr.Code {
+	case foreignKeyErrorCode:
+		return errors.New(somethingWentWrongError)
```

Comment:
Having so generic message is not so nice and it could create debugging issues down the road 😵 I don't have a better option now

Reply:
I also don't have better solution for this that's why I provided generic message ATM.